include/merlin_hashtable.cuh

```diff
@@ -507,10 +524,20 @@ class HashTable {
    * @param stream The CUDA stream that is used to execute the operation.
    *
    */
-  void find(size_type n, const key_type* keys,  // (n)
-            value_type* values,                 // (n, DIM)
-            bool* founds,                       // (n)
-            meta_type* metas = nullptr,         // (n)
+  inline void find(size_type n, const key_type* keys,  // (n)
```

Comment:
What's the benefit of adding new inline alias for each APIs?

Reply:
1. It is the right pointer format. Because that is the pointer-type used in the function.
2. The code becomes more readable. In particular because I will index into the buffer.
3. You get the flexibility to call without so much casting (if that is your desire).